PWGCF/EbyEFluctuations/Tasks/IdentifiedMeanPtFluctuations.cxx

```diff
@@ -52,162 +53,267 @@ using MyCollision = MyCollisions::iterator;
 using MyTrack = MyTracks::iterator;
 } // namespace o2::aod
 
-struct IdentifiedMeanPtFluctuations {
+double massPi = TDatabasePDG::Instance()->GetParticle(211)->Mass();
+double massKa = TDatabasePDG::Instance()->GetParticle(321)->Mass();
+double massPr = TDatabasePDG::Instance()->GetParticle(2212)->Mass();
+
+// struct IdentifiedMeanPtFluctuations {
+struct MeanpTFluct {
```

Comment:
Why are you changing the name if you are doing identified particles mean pT fluctuations?
The name probably collides with the unidentified particles mean pT fluctuations. Isn't it?

Reply:
> Why are you changing the name if you are doing identified particles mean pT fluctuations? The name probably collides with the unidentified particles mean pT fluctuations. Isn't it?

Sorry I did it by mistake. Actually I was copying a file whose name is MeanpTFluct to IdentifiedMeanPtFluctation.cxx because in my local machine i was running my task under the name MeanpTFluct.
It's a mistake from my side. I will change the name in the task.